InteractiveHtmlBom/ecad/schema/genericjsonpcbdata_v1.schema

```diff
@@ -0,0 +1,574 @@
+                "type": { "type": "string", "const": "segment" },
+                "start": { "$ref": "#/definitions/XY" },
+                "end": { "$ref": "#/definitions/XY" },
+                "width": { "type": "number" }
+            },
+            "minProperties": 4,
+            "title": "DrawingSegment"
+        },
+        "DrawingRect": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "type": { "const": "rect" },
+                "start": { "$ref": "#/definitions/XY" },
+                "end": { "$ref": "#/definitions/XY" },
+                "width": { "type": "number" }
+            },
+            "minProperties": 4,
+            "title": "DrawingRect"
+        },
+        "DrawingCircle": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "type": { "const": "circle" },
+                "start": { "$ref": "#/definitions/XY" },
+                "radius": { "type": "number" },
+                "width": { "type": "number" }
+            },
+            "minProperties": 4,
+            "title": "DrawingCircle"            
+        },
+        "DrawingArc": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "type": { "const": "arc" },
+                "width": { "type": "number" },
+                "svgpath": { "type": "string" },
+                "start": { "$ref": "#/definitions/XY" },
+                "radius": { "type": "number" },
+                "startangle": { "type": "number" }, 
+                "endangle": { "type": "number" }
+            },
+            "required": [
+                "type",
+                "width",
+                "svgpath"
+            ],
+            "title": "DrawingArc"            
+        },
+        "DrawingCurve": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "type": { "const": "curve" },
+                "start": { "$ref": "#/definitions/XY" },
+                "end": { "$ref": "#/definitions/XY" },
+                "cpa": { "$ref": "#/definitions/XY" },
+                "cpb": { "$ref": "#/definitions/XY" },
+                "width": { "type": "number" }
+            },
+            "minProperties": 6,
+            "title": "DrawingCurve"            
+        },
+        "DrawingPolygon": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "type": { "const": "polygon" },
+                "svgpath": { "type": "string" },
+                "pos": { "$ref": "#/definitions/XY" },
+                "angle": { "type": "number" }, 
+                "polygons": { 
+                    "type": "array",
+                    "items": { 
+                        "type": "array",
+                        "items": { "$ref": "#/definitions/XY" }
```

Comment:
restrict size to 2

Reply:
The definition for "Coordinates" (formerly "XY") has size restricted to (exactly) 2.  
This comment seems to be saying that the array of coordinates in a polygon should be restricted to size 2, which conflicts with DATAFORMAT.md